tests/test_core/test_org_backfill_data.py

```
@@ -0,0 +1,292 @@
+    ).fetchone()
+    assert row[0] == org_id, f"workflow_metrics org_id mismatch: {row[0]!r} != {org_id!r}"
+
+
+def test_backfill_aggregator_returns_correct_shape(_shared, monkeypatch):
+    """backfill_data_org_ids() returns {"test_runs", "llm_traces", "workflow_metrics"}.
+
+    Patches source-module accessor symbols so the aggregator uses the isolated
+    instances — the lazy `from X import fn` inside the function picks up the
+    patched symbol from sys.modules at call time.
+    """
+    registry, store, collector, admin, _dsn = _shared
```

Ruff (0.15.18)

[warning] Unpacked variable `admin` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)